lib/baldr/grammar/version4010/set310.rb

```diff
@@ -35,13 +35,14 @@ def record_defs
           {id: 'M7', min: 0, max: 5}
         ]},
        {id: 'L0', min: 0, max: 120, level:[
-         {id: 'L5', min: 0, max: 9999},
-       ]},
+          {id: 'L5', min: 0, max: 9999},
```

Comment:
@midhunkrishna Can you please remove this comma here, I think I put that on accident last PR.